src/vorta/notifications.py

```diff
@@ -1,6 +1,11 @@
 import sys
 from vorta.models import SettingsModel
 
+if sys.platform == 'darwin':
```

Comment:
If dbus isn't available, the import fails. Since Linux is very heterogeneous, we have to assume it fails.

So I moved those imports to a try-except block.